### PR TITLE
add: an user option for padding

### PR DESCRIPTION
**Pull request by AmaiKinono**
I found a 1-character padding is enough for me, so I created this patch.

### CODE REVIEW

File: mini-modeline.el [hunk 1]
```diff
@@ -94,6 +94,10 @@ Nil means current selected frame."
   :type 'sexp
   :group 'mini-modeline)
 
+(defcustom mini-modeline-right-padding 3
+  "Padding to use in the right side.
+Set this to the minimal value that doesn't cause truncation.")
```
Comment (kiennq) [inline mini-modeline.el]: Can you add `:type` and `:group` to this `defcustom`? So user can easily find this via UI

Comment (AmaiKinono) [reply mini-modeline.el]: Done.

**Comment by kiennq** [comment]
Thanks for your contribution 😄

**Comment by kiennq** [comment]
I've chosen squashed merge but it somehow remove your name from it, will work on getting back your name as author

**Comment by AmaiKinono** [comment]
Thanks for your kindness :smile: I actually don't care as long as I can use the patch.

**Comment by kiennq** [comment]
Your name's on now 😄

**Pull request merged**